spring-cloud-gateway-core/src/test/java/org/springframework/cloud/gateway/filter/factory/SetStatusGatewayFilterFactoryTests.java

```diff
@@ -39,9 +41,6 @@
 import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.context.junit4.SpringRunner;
 
-import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Please move these back as the PR compile is failing on checkstyle location for the static imports.

Reply:
Opps. change it now.